eZ/Publish/API/Repository/Values/Filter/Filter.php

```diff
@@ -143,6 +143,20 @@ public function withSortClause(FilteringSortClause $sortClause): Filter
         return $this;
     }
 
+    public function withOffset(int $offset): Filter
```

Comment:
We don't use it in the other methods, so I don't mind, but why not 

```suggestion
    public function withOffset(int $offset): self
```

? :smile:

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/210/commits/7f792aee07fdec45bac26e42c7e56a67de6a4aeb